coverage.py

```diff
@@ -156,9 +171,16 @@ def run_on_specific_identifiers(self, identifiers):
             index += self.batch_size
         return (successes, transient_failures, persistent_failures), records
 
-    def run_once(self, offset):
-        batch = self.items_that_need_coverage().limit(
-            self.batch_size).offset(offset)
+    def run_once(self, offset, covered_statuses=None):
+        covered_statuses = covered_statuses or BaseCoverageRecord.DEFAULT_COUNTS_AS_COVERED
+        # Make it clear which class of items we're covering on this
+        # run.
```

Comment:
I understand what this is doing, but the comment sounds wrong to me - covered_statuses is the class of items we're _not_ covering on this run, because they are already considered covered. Maybe it would help to think of a less ambiguous variable name?